template/8/http/index.php

```diff
@@ -0,0 +1 @@
+../../7/http/index.php
```

Comment:
Is this a symbolic link? Could it have issues with Windows NTFS filesystem?

Reply:
This is indeed a symbolik and I didn't thought about windows compatibility to be honest.

Reading some things about this online, it appears it could cause issues.

Maybe the simplest solution is to duplicate these files, they are small enough and won't be changed often (as they are mostly meant to be used for demo and introduction to bref).

Reply:
👍  let's be safe then and duplicate (no idea about Windows too)